src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2811,9 +2871,29 @@ There are two configuration options to control this mechanism (present in the `l
 ** The pa.ldap.tenant.attr configuration is not defined.
 ** The pa.ldap.tenant.attr configuration is defined, but the configured attribute is empty for the user.
 
+==== Shadow Accounts
+
+Shadow accounts are an optional feature that can be enabled in the LDAP configuration file.
+
+When enabled, whenever a user logs in successfully using LDAP credentials, an internal shadow account with a randomly generated password will be created.
```

Comment:
If the feature was false, we log in, then we set it to true, does it remove the previously stored LDAP credential ?

Reply:
it will not. e.g. jobs that were already submitted by the user have the user credentials attached